osl/maxfilter/maxfilter.py

```diff
@@ -573,13 +576,21 @@ def main(argv=None):
             sys.exit('Trans file not found ({0})'.format(args.trans))
 
     print('Processing {0} files'.format(sum(good_fifs)))
+    if args.outdir == None:
+        args.outdir = 'None'
+    print('we came to 0')
     if args.outdir == 'adjacent':
         print('Outputs will be saved alongside inputs\n\n')
     else:
-        if os.path.isdir(args.outdir) is False:
-            sys.exit('Output directory not found ({0})'.format(args.trans))
+        if '{' in args.outdir and '}' in args.outdir:
+            print('we came to 1')
```

Comment:
debugging message?

Reply:
Yep

Reply:
Fixed in https://github.com/OHBA-analysis/oslpy/pull/48/commits/600ab9032269cc4e12ed28c8d5171ba0112de2cb